.dockerignore

```diff
@@ -1,4 +1,5 @@
 .git
 firecracker
-target
+target/debug
```

Comment:
Why do we need to change this?

Reply:
The Dockerfile is now copying `target/release/{kani,cargo-kani}` into the container.

Reply:
Updated: Now `cargo install`ing the package, so we ignore `target/{release,debug}`, but need `target/package`